README.rst

```diff
@@ -242,6 +242,43 @@ Run qa tests:
 
     ./run-qa-checks
 
+Quickstart Guide
+----------------
+
```

Comment:
I think we need to spend a few words summarizing how this module works.
Some users somehow think that the module is magic and they expect it to do everything automatically (LOL), so I think it's good to summarize here how the module works, eg:

This module works by periodically collecting the network topology graph data from the supported softwares (link current features / supported formats). The data has to be either fetched by the application or received in POST API requests, therefore after deploying the application additional steps are required to make the data collection and visualization work, read on to find out how.